manifests/pki/ca.pp

```diff
@@ -1,30 +1,32 @@
-# Class: strongswan::pki::ca
+# @summary This class manages the Strongswan CA
 #
-# This module manages Strongswan CA
+# @param common_name
+#   The Certificate Authority `Common Name (CN)`
+# @param country_code
+#   The Certificate Authority `Country Code (C)`
+# @param organization
+#   The Certificate Authority `Organization (O)`
 #
-# Example configuration:
-# =======================
+# @example Default CA configuration
+#   include strongswan::pki::ca
 #
-# #Default:
-# include strongswan::pki::ca
-#
-#
-# #With Params:
-# class {'strongswan::pki::ca':
-#   $common_name     => 'myVPN',
-#   $country_code    => 'XX',
-#   $organization    => 'myOrg',
-# }
-
+# @example Configure the CA with custom options
+#   class {'strongswan::pki::ca':
+#     $common_name  => 'myVPN',
+#     $country_code => 'XX',
+#     $organization => 'myOrg',
+#   }
 class strongswan::pki::ca (
-  $ca_certificate_dir = $strongswan::ca_certificate_dir,
```

Comment:
The parameters that don't really form part of the class's API have been moved into the body.  This makes the reference documentation cleaner.